src/components/data_search/DatasetSearchTable.jsx

```diff
@@ -49,8 +51,17 @@ export const DatasetSearchTable = (props) => {
   const [selectedTable, setSelectedTable] = useState(datasetSearchTableTabs.study);
   const [searchTerm, setSearchTerm] = useState('');
 
-  const isFiltered = (filter, category) => (filters[category]).indexOf(filter) > -1;
-  const numSelectedFilters = (filters) => Object.values(filters).reduce((sum, array) => sum + array.length, 0);
+  const isFilteredArray = (filter, category) => (filters[category]).indexOf(filter) > -1;
+
+  const anyFiltersSelected = (filters) => {
+    return Object.values(filters).some((filter) => {
+      if (isArray(filter)) {
+        return filter.length > 0;
+      } else {
+        return filter !== null;
+      }
```

Comment:
Not sure if it's clearer but you can express this using a single `return`, at which point this can also be written without the `{ }`s.
```suggestion
      return isArray(filter) ? filter.length > 0 : filter !== null
```

Actually I think lodash `isEmpty()` is equivalent to `filter.length > 0 || filter != null` so this should work
```
const anyFiltersSelected = (filters) => 
  Object.values(filters).some(filter => !isEmpty(filter));
```

Reply:
nice!

Reply:
isEmpty doesn't work for this case, but the suggested change is still much cleaner!